pytorch_impl/losses.py

```
@@ -0,0 +1,51 @@
+import torch
+from torch import nn
+import torch.nn.functional as F
```

Ruff (0.11.9)

`torch.nn.functional` imported but unused

Remove unused import: `torch.nn.functional`

(F401)